src/RiakClient/Models/MapReduce/RiakMapReduceResultPhase.cs

```diff
@@ -28,10 +28,10 @@ namespace RiakClient.Models.MapReduce
     public class RiakMapReduceResultPhase
     {
         private readonly bool success;
-        private readonly uint phase;
+        private readonly long phase;
```

Comment:
I would hope we don't ever have somebody run a MR job with 2^16 phases though.... int *should* be fine :)